### PR TITLE
Network spec fix more broken links, add line breaks for ease of future updates

### PR DESCRIPTION
**Pull request by gregnazario**
Network spec fix more broken links.  Reviewed TODOs

### CODE REVIEW

**Review by bmwill — DISMISSED** [review]

**Comment by gregnazario** [comment]
Had to rebase, docker-compose tests failed

**Review by davidiw — APPROVED** [review]
gogogo

/land

File: specifications/network/README.md [hunk 1]
```diff
@@ -6,42 +6,71 @@
...
+This document describes **DiemNet**, the primary network protocol used for communication between any two nodes in the
+Diem ecosystem. The protocol only describes the structure and order of messages on the wire, while relying on an
+underlying transport (TCP) for the actual message delivery. In addition, all communication between peers MUST be
+encrypted and authenticated using [Noise protocol](noise.md). DiemNet supports multiplexing several application
+protocols concurrently over a single connection. For each application protocol, DiemNet provides two kinds of messaging
+semantics: (1) a fire-and-forget DirectSend and (2) a unary RPC.
```
[the remark this answers is not in the document]
Comment (gregnazario) [reply specifications/network/README.md]: I guess I believed too much in the character line limit :P

**Review by davidiw — APPROVED** [review]
/land

**Pull request merged**